test/models/user_test.rb

```diff
@@ -4,4 +4,11 @@ class UserTest < ActiveSupport::TestCase
   test "factory" do
     assert build(:user).validate!
   end
+
+  test "should not allow duplicate email addresses" do
```

Comment:
Just a suggestion but I find that removing words like `should` make test descriptions easier to parse. 

`test "validate uniqueness of email"`